web/app/themes/radcliffe/searchform.php

```diff
@@ -1,4 +1,5 @@
-<form method="get" class="search-form" action="<?php echo esc_url( home_url( '/' ) ); ?>">
-	<input type="search" placeholder="<?php _e( 'Search form', 'radcliffe' ); ?>" name="s" id="s" /> 
-	<input type="submit" id="searchsubmit" value="<?php _e( 'Search', 'radcliffe' ); ?>" class="search-button">
+<form id="search2" class="search-form" method="get" action="<?php echo esc_url(home_url('/')); ?>">
```

Comment:
Please replace locale functions and placeholder attributes.

Reply:
See above comments re placeholders and the complexity of the search instances.

Reply:
Thank you for the clarification Stephanie. I totally agree with you that we should have the text remaining - for now let's keep the placeholders out.